### PR TITLE
[Model]Implentation of MERIT(IJCAI2021)

### PR DESCRIPTION
**Pull request by zhengziyu77**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR

### CODE REVIEW

File: examples/merit/merit_trainer.py [hunk 1]
```diff
@@ -1,85 +1,66 @@
...
 from gammagl.datasets.amazon import Amazon
+
 class Unsupervised_Loss(WithLoss):
     def __init__(self, net):
         super(Unsupervised_Loss, self).__init__(backbone=net, loss_fn=None)
 
     def forward(self, data, label=None):
-        #loss = self._backbone(data['adj_1'], data['adj_2'], data['aug_feat1'], data['aug_feat2'], data['sparse'])
-        loss = self._backbone(data['graph1'], data['graph2'])
+        loss = self._backbone(data['feat1'], data['edge1'], data['weight1'],data['num_node1'],
```
Comment (zsy0828) [inline examples/merit/merit_trainer.py:26]: label随便设置一个，但不能没有

File: examples/merit/merit_trainer.py [hunk 2]
```diff
@@ -1,85 +1,66 @@
...
-    A = (sparse_adj + sp.eye(num_node)).tocoo()#原始图的邻接矩阵
-    diff = gdc(A, alpha=args.alpha, eps=0.0001)
 
-    norm_diff = sp.csr_matrix(diff).tocoo()
-    diff_row, diff_col,diff_weight = norm_diff.row,norm_diff.col,norm_diff.data
-    diff_edge_index =tlx.convert_to_tensor([diff_col,diff_row],dtype='int64')
-    diff_weight=tlx.convert_to_tensor(diff_weight,dtype='float32')
+    weight = np.ones(edge_index.shape[1])
+    sparse_adj = sp.coo_matrix((weight, (edge_index[0], edge_index[1])), shape=(num_node, num_node))
+    A = (sparse_adj + sp.eye(num_node)).tocoo()#原始图的邻接矩阵
```
Comment (zsy0828) [inline examples/merit/merit_trainer.py:53]: 这里不就是加一个自环操作么，这里不能用add_self_loop方法么

File: examples/merit/process.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import scipy.sparse as sp
+import numpy as np
+import tensorlayerx as tlx
+def compute_diff(A: sp.csr_matrix, alpha: float, eps: float):
+    N = A.shape[0]
+    A_loop = sp.eye(N) + A
```
Comment (zsy0828) [inline examples/merit/process.py:6]: 我看你传进来的A就是一个带自环的了，这里还要再加么

File: examples/merit/merit_trainer.py (hunk 1 above)
Comment (Theheavens) [reply examples/merit/merit_trainer.py:26]: 看起来下面已经传了，把`=None`去掉吧。

File: examples/merit/REMADE.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+```
+
+## 	Performance
+
+
+|    Dataset    | Cora | Citeseer | Pubmed |
+| :-----------: | :--: | :------: | :----: |
+| Author's Code | 83.1 |   74.0   |  80.2  |
+|    GammaGL    | 84.3 |   72.2   |  --.-  |
```
Comment (Theheavens) [inline examples/merit/REMADE.md:58]: 指明一下这是什么后端的结果。每个后端都列出来，还没有结果的先空着

**Pull request closed without merging**